### PR TITLE
Added login and AuthenticationFilter

### PR DESCRIPTION
**Pull request by aleksandr-hrankin**

### CODE REVIEW

**Review by Rommelua — APPROVED** [review]

**Comment by Rommelua** [comment]
well done

File: src/main/java/com/internet/shop/web/filter/AuthenticationFilter.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import javax.servlet.http.HttpServletResponse;
+
+public class AuthenticationFilter implements Filter {
+    private static final String USER_ID = "user_id";
+    private static final Injector injector = Injector.getInstance("com.internet.shop");
+    private UserService userService = (UserService) injector.getInstance(UserService.class);
+
+    @Override
+    public void init(FilterConfig filterConfig) throws ServletException {
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/web/filter/AuthenticationFilter.java]:
```suggestion
```

File: src/main/java/com/internet/shop/web/filter/AuthenticationFilter.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        if (userId == null || userService.get(userId) == null) {
+            resp.sendRedirect("/login");
+            return;
+        }
+        filterChain.doFilter(req, resp);
+    }
+
+    @Override
+    public void destroy() {
+
```
Comment (dmytro-hryhoruk) [inline src/main/java/com/internet/shop/web/filter/AuthenticationFilter.java:54]:
```suggestion
```

**Review by dmytro-hryhoruk — APPROVED** [review]
Good job!

**Pull request merged**